src/core.js

```diff
@@ -24,17 +24,24 @@ $.extend($.fn, {
 
 		if ( validator.settings.onsubmit ) {
 
-			this.validateDelegate( ":submit", "click", function( event ) {
+			this.validateDelegate( ":submit, :submit *", "click", function( event ) {
 				if ( validator.settings.submitHandler ) {
 					validator.submitButton = event.target;
 				}
+
+				// locate the actual "action" element
+				var actionTarget = $(event.target);
```

Comment:
Right, but I want to achieve it in a different way as implemented currently. Instead of binding to `:submit *` we should do a find only when event.target doesnt work.

This would help because only chrome needs to pay for this edge case, not all browsers